Extensions/Notification/JsExtension.js

```diff
@@ -0,0 +1,185 @@
+        //'JsPlatform/Extensions/'
+      )
+      .addParameter('expression', _('Id'), '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Notification/notificationtools.js')
+      .setFunctionName('gdjs.evtTools.notification.onShow');
+
+    extension
+      .addCondition(
+        'IsClosed',
+        _('Notification has been closed'),
+        _('Check if the notification has been closed'),
+        _('Check if notification with an id of _PARAM0_ has been closed'),
+        _('Notification'),
+        //'JsPlatform/Extensions/',
+        //'JsPlatform/Extensions/'
+      )
+      .addParameter('expression', _('Id'), '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Notification/notificationtools.js')
+      .setFunctionName('gdjs.evtTools.notification.onClose');
+
+    extension
+      .addCondition(
+        'IsError',
+        _('Notification has an error'),
+        _('Check if an error happend with the notification'),
+        _('Check if notification with an id of _PARAM0_ has an error'),
+        _('Notification'),
+        //'JsPlatform/Extensions/',
+        //'JsPlatform/Extensions/'
+      )
+      .addParameter('expression', _('Id'), '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Notification/notificationtools.js')
+      .setFunctionName('gdjs.evtTools.notification.onError');
+
+    extension
+      .addAction(
+        'RequestPermission',
+        _('Request permissions for notifications'),
+        _('Request permissions for notifications from the user'),
+        _('Request permission for notifications from the user'),
+        _('Notification'),
+        //'JsPlatform/Extensions/',
+        //'JsPlatform/Extensions/'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Notification/notificationtools.js')
+      .setFunctionName('gdjs.evtTools.notification.requestPermission');
+
+    extension
+      .addAction(
+        'CreateNotification',
+        _('Create a new notification'),
+        _('Create a new notification (always trigger once)'),
+        _('Create a new notification with an id of _PARAM0_, a title of _PARAM1_ and a message of _PARAM2_'),
+        _('Notification'),
+        //'JsPlatform/Extensions/',
+        //'JsPlatform/Extensions/'
+      )
+      .addParameter('expression', _('Id'), '', false)
+      .addParameter('string', _('Title'), '', false)
+      .addParameter('string', _('Message'), '', false)
+      .addParameter('string', _('(Optional) file location of an image to display'), '', true)
+      .addParameter('string', _('(Optional) change language of text, must be an ISO 2 Letter Language Code'), '', true)
+      .setDefaultValue('false')
+      .addParameter('yesorno', _('(Optional) require interaction to make notification disappear'), '', true)
+      .setDefaultValue('false')
+      .addParameter('yesorno', _('(Optional) notification is silent'), '', true)
```

Comment:
The `setDefaultValue` should be after the `addParameter`
```suggestion
      .addParameter('yesorno', _('(Optional) require interaction to make notification disappear'), '', true)
      .setDefaultValue('false')
      .addParameter('yesorno', _('(Optional) notification is silent'), '', true)
      .setDefaultValue('false')
```